plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1471,6 +1474,9 @@ function( $subscription ) use ( $site_id ) {
 				$local                              = array_shift( $local );
 				$subscription['local']['installed'] = true;
 				$subscription['local']['version']   = $local['Version'];
+				$subscription['local']['type']      = $local['_type'];
+				$subscription['local']['path']      = $local['_filename'];
+				$subscription['local']['slug']      = null;
```

Comment:
Is this what's causing the slug to be blank in my API Body?

Reply:
I suspect the same thing: https://github.com/woocommerce/woocommerce/pull/40752#discussion_r1366620881

In the past commits, this was also `null` but then later set to something else. [We have lost these lines with the merge commit](https://github.com/woocommerce/woocommerce/pull/40752/commits/aeae2fe73d98f150fa2d7bcfafc147a2ccf18458#diff-32b26bf8430b272079f3c981317024f2a7d7848b4198ae7878716cc133d10961L1482). I suspect something went wrong with the merge.

Reply:
Thanks @raicem! I added the code back and checked the rest of the merge to make sure nothing else is missing.